projects/1.20/assets/powah-rearchitected/powah/lang/zh_cn.json

```
@@ -0,0 +1,265 @@
+    "item.powah.lens_of_ender": "末影透镜",
+    "item.powah.niotic_crystal": "钻石晶体",
+    "item.powah.photoelectric_pane": "光伏板",
+    "item.powah.player_aerial_pearl": "玩家天线珍珠",
+    "item.powah.spirited_crystal": "富生晶体",
+    "item.powah.steel_energized": "充能钢",
+    "item.powah.thermoelectric_plate": "热力板",
+    "item.powah.uraninite": "晶质铀",
+    "item.powah.uraninite_raw": "粗晶质铀",
+    "item.powah.wrench": "扳手",
+    "itemGroup.powah.tab": "Powah",
+    "jei.powah.binding_card_dim": "对末影人或末影螨使用绑定卡获得。",
+    "jei.powah.lens_of_ender": "对末影人或末影螨使用光伏板获得。",
+    "jei.powah.player_aerial_pearl": "对僵尸或尸壳使用天线珍珠以获得。",
+    "tag.item.c.blue_ices": "蓝冰",
+    "tag.item.c.dry_ices": "干冰",
+    "tag.item.c.ices": "任意冰",
+    "tag.item.c.packed_ices": "浮冰",
+    "tag.item.c.raw_uraninite_ores": "粗晶质铀矿石",
+    "tag.item.c.regular_ices": "冰",
+    "tag.item.c.uraninite_blocks": "晶质铀块",
+    "tag.item.c.uraninite_ores": "晶质铀矿石",
+    "wiki.powah.battery": "电池",
+    "wiki.powah.battery_0": "置于物品栏中时充能其他物品。在末影单元的GUI中，Shift单击电池可为末影网络的频道提升储量。若电池中存有能量，会被加入末影网络的频道。",
+    "wiki.powah.binding_card": "绑定卡",
+    "wiki.powah.binding_card_0": "绑定卡用于链接位于同一维度的玩家与玩家供电仪。",
+    "wiki.powah.binding_card_1": "用于玩家供电仪之前，需将其与自己绑定，右击以进行绑定。",
+    "wiki.powah.binding_card_dim": "绑定卡（跨维度）",
+    "wiki.powah.binding_card_dim_0": "跨维度绑定卡用于跨维度链接玩家与玩家供电仪。",
+    "wiki.powah.binding_card_dim_1": "用于玩家供电仪之前，需将其与自己绑定，右击以进行绑定。",
+    "wiki.powah.charged_snowball": "带电雪球",
+    "wiki.powah.charged_snowball_0": "落地或命中生物时会召唤闪电。充能雪球以获得。",
+    "wiki.powah.dry_ice": "干冰",
+    "wiki.powah.dry_ice_0": "干冰主要用于降低反应堆的温度。它生成于Y=64以下，但同时也可以通过充能两个蓝冰获得。",
+    "wiki.powah.ender_cell": "末影单元",
+    "wiki.powah.ender_cell_0": "末影单元是存储存储能量（FE）的方块，其中能量专属与方块主人的特定末影网络频道。",
+    "wiki.powah.ender_cell_1": "只要用于能量传输的末影单元拥有处于激活状态且容量有效的频道，你就可以在任何位置访问该频道中储存的能量。",
+    "wiki.powah.ender_cell_2": "你可以在末影单元GUI中，按Shift单击能量单元或电池，来将其添加至所选频道。",
+    "wiki.powah.ender_cell_3": "增加的能量容量与被添加的物品相同，频道的最大能量容量为9*10^18 FE。",
+    "wiki.powah.ender_gate": "末影道门",
+    "wiki.powah.ender_gate_0": "在相邻方块与末影网络之间传输能量。",
+    "wiki.powah.ender_gate_1": "与末影单元不同，你不能通过末影道门升级各个频道。",
+    "wiki.powah.energizing": "充能",
+    "wiki.powah.energizing_0": "充能台用于充能物品，周围%s格范围内至少需要存在一台充能棒才能工作，充能速度取决于充能棒的数量和等级（能量传输速率）。",
+    "wiki.powah.energizing_1": "充能台本身不需要能量供应，但充能棒必须放置在线缆或储能（FE）方块上才能工作。",
+    "wiki.powah.energizing_2": "放置后，同一范围内的充能台和充能棒会相互自动链接，也可使用链接模式下的<powah:wrench>来手动重新链接。",
+    "wiki.powah.energy_blocks": "功能性方块",
+    "wiki.powah.energy_cable": "能量线缆",
+    "wiki.powah.energy_cable_0": "线缆用于在机器间传输能量。",
+    "wiki.powah.energy_cable_1": "你可通过使用配置模式下的<powah:wrench>右击线缆来改变其传输模式。",
+    "wiki.powah.energy_cell": "能量单元",
+    "wiki.powah.energy_cell_0": "能量单元可存储能量（FE）。",
+    "wiki.powah.energy_cell_1": "也可在末影单元GUI中，通过Shift单击将其添加到选定末影网络频道中，为其增加能量容量，其中存储的能量会一并添加到频道中。",
+    "wiki.powah.energy_discharger": "能量提取器",
+    "wiki.powah.energy_discharger_0": "能量提取器可提取含能物品中的能量（FE），并存储在自身缓存中。使用线缆连接能量提取器以再次利用这些能量。",
```

| **审阅级别** |
|:--------------:|
|⚪ **轻微** / 不阻止合并但需三思|

原文：
- **The Energy Discharger its a block used to drain energy (FE) out of charged items and then store it to an internal buffer if then connected via cables to extract that stored power and re-using it again.**
```suggestion
    "wiki.powah.energy_discharger_0": "能量提取器可提取含能物品中的能量（FE），并存储在自身缓存中。之后，如果通过线缆连接，就可以从中提取这些储存的能量并重新使用。",
```